### DISCUSSION TITLE
System Update - Check for update failure !

### DISCUSSION THREAD

**Discussion by gfracasso** [LingmoOS/Community#27] [Bugs Report/缺陷提交]
When I start System Update, I receive the following message:

Repository 'http://packages.lingmo.org/lingo hydrogen InRelease' chanced its ?Origin? value from " to 'LingmoOS'.

Is there anybody can help me about this problem ?

My ber regards
Giovanni from Italy

**Comment by elysia-best** [comment]
@lingmo-dream have performed some upgrade to the repo for Hydrogen version. They should be upgraded to Helium version by installing the base-files package.

It's okay to ignore this notice once the sources are upgraded.
